mmv1/third_party/terraform/tests/resource_gke_hub_feature_iam_test.go.erb

```diff
@@ -0,0 +1,310 @@
+<% autogen_exception -%>
+package google
+
+import (
+	"fmt"
+	"testing"
+
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/resource"
+
+	"github.com/hashicorp/terraform-provider-google/google/acctest"
+)
+
+func TestAccGKEHub2FeatureIamBindingGenerated(t *testing.T) {
+	t.Parallel()
```

Comment:
```suggestion
	// VCR fails to handle batched project services
	acctest.SkipIfVcr(t)
	t.Parallel()
```
Looks like we need to skip these tests from VCR as well. Sorry for missing this earlier